ext/symengine/symengine.c

```diff
@@ -57,6 +57,9 @@ void Init_symengine()
     // Sympify as a Module Level Function
     rb_define_module_function(m_symengine, "convert", cutils_sympify, 1);
     rb_define_global_function("SymEngine", cutils_sympify, 1);
+    
+    // evalf as a Module Level Function
+    rb_define_module_function(m_symengine, "evalf", cutils_evalf, 3);
```

Comment:
Currently you need to give all 3 parameters. Can you change it so that only 1 parameter is required and the other 2 have default values `53` and `False`